aws_batch/pipeline.py

```diff
@@ -357,6 +359,44 @@ def generate_taxid_annotated_m8(input_m8, output_m8, accession2taxid_db):
             outf.write(new_line)
     outf.close()
 
+def read_columnfile_into_list(file_name):
+    with open(file_name) as f:
+        content = f.readlines()
+    return [x.strip("\n") for x in content]
+
+def filter_deuterostomes_from_m8(input_m8, output_m8, deuterostome_file):
+    taxids_toremove = read_columnfile_into_list(deuterostome_file)
```

Comment:
i will just call the function name read_file_into_list?

Reply:
makes sense